docs/Getting_Started.md

```diff
@@ -2,106 +2,116 @@
-      * `./hokusai/development.yml`- a Docker Compose YAML file for the development environment
-      * `./hokusai/test.yml` - a Docker Compose YAML file for the test environment
-      * `./hokusai/staging.yml` - a Kubernetes YAML file for the staging environment
-      * `./hokusai/production.yml` - a Kubernetes YAML file for the production environment
-  - An ECR repository for your project
+Note: If you set the environment variable `AWS_ACCOUNT_ID` in your shell, you can omit the `--aws-account-id` option.
 
-  The files in `./hokusai` as well as the `Dockerfile` are meant to be a starting point for development of your specific application's dependencies, and can / should be freely modified, as you customize your Docker build, add service dependencies to your environments, introduce environment variables, or change the container runtime commands.
+`hokusai setup` will create:
+- A `Dockerfile` in your project root
+- A configuration folder `./hokusai`.  This folder contains:
+  * `./hokusai/config.yml` - Hokusai project configuration
+  * `./hokusai/common.yml` - a Docker Compose YAML file imported by `development.yml` and `test.yml`
+  * `./hokusai/development.yml`- a Docker Compose YAML file for the development environment
+  * `./hokusai/test.yml` - a Docker Compose YAML file for the test environment
+  * `./hokusai/staging.yml` - a Kubernetes YAML file for the staging environment
+  * `./hokusai/production.yml` - a Kubernetes YAML file for the production environment
+- An ECR repository for your project
+
+The files in `./hokusai` as well as the `Dockerfile` are meant to be a starting point for development of your specific application's dependencies, and can / should be freely modified, as you customize your Docker build, add service dependencies to your environments, introduce environment variables, or change the container runtime commands.  See [Configuration Options](./Configuration_Options.md) if you want to modify your project's configuration.
 
 2) Check that Hokusai is correctly configured for your project
 
-  `hokusai check` will run a series of tests and warn you if it finds anything improperly configured.
+`hokusai check` will run a series of tests and warn you if it finds anything improperly configured.
 
-3) Run the development stack
+3) Run the development environment
 
-  `hokusai dev start` will build a Docker image, then start a Docker Compose stack defined by `./hokusai/development.yml`
+`hokusai local dev start` will build a Docker image, then start a Docker Compose environment defined by `./hokusai/development.yml`
 
-  If this command throws no errors, try interacting with the running stack with:
+If this command throws no errors, try interacting with the running environment with:
 
-  ```bash
-  hokusai dev status
-  hokusai dev logs
-  hokusai dev shell
-  ```
+```bash
+hokusai local dev status
+hokusai local dev logs
+hokusai local dev run bash
```

Comment:
one strange thing about the command structure (both before and after this PR), is that environments are top level commands in the `local` context but are flags in the remote context (e.g. `dev` here verses `--production` for remote commands)

Reply:
i don't have a better suggested structure, this is just an observation.

Reply:
if we were to be consistent i guess we would do: 

```
hokusai dev [CMD]
hokusai test
hokusai production [CMD]
hokusai staging [CMD]

Reply:
The issue with my suggested structure is that non environment specific commands (like `promote`) would live at the top-level.

Reply:
Yup. This was exactly my line of reasoning, and it's because I didn't want to hoist the promote command to the top level that I didn't try making `staging` and `production` their own command groups. I could give it a go however, and maybe with a bit of metaprogramming get a syntax like `hokusai promote staging to production` WDYT?

Reply:
I think it's pretty intuitive that the `deployment` namespace refers to a remote context actually, I don't have a problem with that living in the top-level. In the structure I'm proposing, you might consider putting `promote` under `staging` as `hokusai staging promote`, but I also think it's fine to group it with the other deployment sub-commands as you do now and have `deployment` live at the top level.

My main issue was with our use of environments as both options and top-level command namespaces and the inconsistency arising from that.